config/initializers/timeout.rb

```diff
@@ -1 +0,0 @@
-Rack::Timeout.service_timeout = ENV.fetch('RACK_SERVICE_TIMEOUT', 12).to_i # seconds
```

Comment:
would appreciate context on this and everything below, thanks!

Reply:
we upgraded `rack-timeout` gem and `service_timeout=` is deprecated for some reason, I didn't spend more time finding the correct way to specify it since the default is 15s. If you think it's important to use 12s instead of default, I'll add it in. https://stackoverflow.com/questions/2583166/set-rails-request-timeout-execution-expired

Reply:
I have no idea why I set it to 12. I think it might have been to mitigate the fact that we're on the Unicorn server, which is not multi-threaded so is vulnerable to slow client attacks.

Reply:
Oh here it is https://github.com/PatrickF1/GraceTunes/pull/61/files. Yeah 15s is good, thanks